_includes/footer.html

```diff
@@ -132,48 +132,38 @@ <h4>Contact us</h4>
     <div class="usa-identifier__container">
```

Comment:
I think it may make sense to pull the identifier into its own component and `include` it in the footer.